File: Cpp/26. Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int index = 0;
        for(int i =1; i < nums.size();i++){
            if(nums[i] != nums[index]){
                index++;
                nums[index] =  nums[i];
            }
        }
        nums.erase(nums.begin() + index +1, nums.end());
        return nums.size();
    }
};

File: Python/26. Remove Duplicates from Sorted Array.py
class Solution(object):
    def removeDuplicates(self, nums):
        nums.sort()
        index = 0
        for i in range(1,len(nums)):
            if nums[i] != nums[index]:
                index += 1
                nums[index] = nums[i]
        del nums[index+1:]
        return len(nums)
